hack/release.toml

```diff
@@ -55,15 +55,15 @@ Any kernel argument that starts with a `-` is dropped. Kernel arguments to be dr
     [notes.updates]
         title = "Component Updates"
         description = """\
-Linux: 6.1.65
+Linux: 6.1.67
 containerd: 1.7.10
```

Comment:
not bumping containerd?

Reply:
yep, I think it's too late - we haven't had any time with it in `main` - I scheduled it for 1.6.1